src/witan/httpapi/components/activities.clj

```diff
@@ -75,18 +76,26 @@
       (nil? receipt)                                      [404 nil nil]
       (not= (:kixi.user/id receipt) (:kixi.user/id user)) [401 nil nil]
       (= "pending" (::spec/status receipt))               [202 nil nil]
-      :else [303 nil {"Location" (::spec/uri receipt)}])))
+      (and (= "complete" (::spec/status receipt))
+           (nil? (::spec/uri receipt)))                   [200 nil nil]
+      (= "complete" (::spec/status receipt))              [303 nil {"Location" (::spec/uri receipt)}])))
```

Comment:
I'm torn about having constants like `(def SEE_OTHER 303)` just so we don't have to look up what the more obscure 3xx and 2xx codes are.

Reply:
Response codes now defined in their own namespace see commit b572b01 for alterations.